From aa6e932978850f4dcf0826cc86c770008140478b Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Wed, 29 May 2013 13:13:17 +0300
Subject: [PATCH 025/290] be (seg.c): comment out unused static functions

Neither seg_header_prepare() nor seg_header_prepare() are used.
Comment them out (don't delete as they may be used by xrgtn).
---
 be/seg.c | 47 ++++++++++++++++++++++++-----------------------
 1 file changed, 24 insertions(+), 23 deletions(-)

diff --git a/be/seg.c b/be/seg.c
index f236c7e..95a7747 100644
--- a/be/seg.c
+++ b/be/seg.c
@@ -40,35 +40,35 @@ struct be_seg_hdr {
 	m0_bcount_t bh_size;
 };
 
-static void seg_header_prepare(struct m0_be_seg *seg, struct m0_stob_io *io)
-{
-	/** fill segment header */
-	/** fill stob ovec: offset and size */
-	/** fill stob ivec: size and addr */
-}
+/* static void seg_header_prepare(struct m0_be_seg *seg, struct m0_stob_io *io) */
+/* { */
+/* 	/\** fill segment header *\/ */
+/* 	/\** fill stob ovec: offset and size *\/ */
+/* 	/\** fill stob ivec: size and addr *\/ */
+/* } */
 
-static int seg_header_write(struct m0_be_seg *seg)
-{
-	struct m0_stob_io io;
-	struct m0_clink   clink;
-	int               rc;
+/* static int seg_header_write(struct m0_be_seg *seg) */
+/* { */
+/* 	struct m0_stob_io io; */
+/* 	struct m0_clink   clink; */
+/* 	int               rc; */
 
-	m0_stob_io_init(&io);
-	seg_header_prepare(seg, &io);
+/* 	m0_stob_io_init(&io); */
+/* 	seg_header_prepare(seg, &io); */
 
-	m0_clink_init(&clink, NULL);
-	m0_clink_add(&io.si_wait, &clink);
+/* 	m0_clink_init(&clink, NULL); */
+/* 	m0_clink_add(&io.si_wait, &clink); */
 
-	rc = m0_stob_io_launch(&io, seg->bs_stob, NULL, NULL);
-	M0_ASSERT(rc != 0); /* XXX FIXME */
+/* 	rc = m0_stob_io_launch(&io, seg->bs_stob, NULL, NULL); */
+/* 	M0_ASSERT(rc != 0); /\* XXX FIXME *\/ */
 
-	m0_chan_wait(&clink);
-	m0_clink_del(&clink);
-	m0_clink_fini(&clink);
-	m0_stob_io_fini(&io);
+/* 	m0_chan_wait(&clink); */
+/* 	m0_clink_del(&clink); */
+/* 	m0_clink_fini(&clink); */
+/* 	m0_stob_io_fini(&io); */
 
-	return io.si_rc;
-}
+/* 	return io.si_rc; */
+/* } */
 
 int m0_be_seg_create(struct m0_be_seg *seg)
 {
@@ -93,6 +93,7 @@ int m0_be_seg_destroy(struct m0_be_seg *seg)
 	M0_PRE(M0_IN(seg->bs_state, (M0_BSS_INIT, M0_BSS_CLOSED)));
 	m0_stob_put(seg->bs_stob);
 	/* XXX TODO: stob destroy ... */
+	return -1; /*XXX*/
 }
 
 void
-- 
1.8.3.2

